crates/swc_ecma_minifier/src/option/terser.rs

```diff
@@ -400,6 +421,11 @@ impl TerserCompressorOptions {
                     })
                 })
                 .collect(),
+            experimental: self.experimental.map(From::from).unwrap_or(
+                CompressExperimentalOptions {
+                    reduce_escaped_newline: self.defaults,
```

Comment:
I think this will enable this optimization by default for most of the users, but I think it's not intended.

Reply:
It's intentional to enable this optimization by default. Because it's already the default value in codegen for the same option.

Reply:
If we disable it by default, how to control the existed minification in codegen?

Reply:
I think it's fine to enable it by default. I just misunderstood your intention.